mongodbatlas/mongodbatlas.go

```diff
@@ -222,7 +204,7 @@ func (resp *Response) CurrentPage() (int, error) {
 
 	pageNum, err := strconv.Atoi(pageNumStr)
 	if err != nil {
-		return 0, fmt.Errorf("error getting current page: %s", err)
+		return 0, fmt.Errorf("error getting current page: %w", err)
```

Comment:
`w` is a go1.13 wrapper for errors which allows `error.Is` to map wrapped errors